addons/project/views/project_views.xml

```diff
@@ -165,11 +165,6 @@
                                     </div>
                                 </div>
                             </div>
-                            <group name="misc">
```

Comment:
Can you check with AJU, I'm not ok with this point (functionnal: how do we set a calendar on the project ? O_0)

Reply:
Ok, she is ok to remove this commit.